docker/local/Dockerfile

```diff
@@ -14,4 +14,4 @@ COPY --from=builder /app/config.toml .
 
 EXPOSE 3005 4005
 
-CMD ["./nitro", "--config", "./config.toml"]
+CMD ./nitro --config ./config.toml
```

Comment:
@bitwiseguy I think in principle we ought to be able to have just the one Dockerfile. The only difference between cloud and local versions (I think) ought to be config. And since we can pass that at runtime, we don't need two different images?

There is one genuine "build time" decision, which is whether to include the UI (the cloud version does it, the local one does not).

Reply:
Yes I agree we should use a single dockerfile and allow the user to configure them differently through the config file if they want.

There is also this [dockerfile](https://github.com/statechannels/go-nitro/blob/main/docker/Dockerfile) which I think is being unused and should be removed. It spins up three go-nitro nodes within the same docker container which seems like an abnormal use-case we probably don't have to support.

Reply:
Yes I agree your proposal is beneficial. 

Let's double check with @kerzhner that he is happy to lose the "three nodes in one container" Dockerfile.

Reply:
The intention with "three nodes in one container" Docker image is to have a very easy to use and simple way for developers to run a sample nitro network. In practice, it is not being used. So we can remove it.